website/docs/docs/dbt-versions/release-notes.md

```diff
@@ -20,6 +20,31 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## October 2024
 
+<Expandable alt_header="Coalesce 2024 announcements">
+
+  Documentation for new features and functionality announced at Coalesce 2024:
+
+  - Iceberg table support for [Snowflake](https://docs.getdbt.com/reference/resource-configs/snowflake-configs#iceberg-table-format)
+  - [Athena](https://docs.getdbt.com/reference/resource-configs/athena-configs) and [Teradata](https://docs.getdbt.com/reference/resource-configs/teradata-configs) adapter support in dbt Cloud.
```

Comment:
sugg parallel construction of bulleted list, some items use periods (.) and some do not